Tests/test_features.py

```diff
@@ -35,6 +35,11 @@ def check_webp_mux(self):
         self.assertEqual(features.check('webp_mux'),
                          _webp.HAVE_WEBPMUX)
 
+    @unittest.skipUnless(HAVE_WEBP, True)
+    def check_webp_anim(self):
```

Comment:
If we want this to run on the CI, which may be useful for debugging, call it test_ instead of check_. Could change the other two too.